src/storage.js

```diff
@@ -0,0 +1,42 @@
+import { getFromLocalStorage, setToLocalStorage } from './functions';
+
+const checkEvent = () => {
+  const checkboxes = document.querySelectorAll('input[type=checkbox]');
+
+  for (let i = 0; i < checkboxes.length; i += 1) {
+    const task = checkboxes[i].nextElementSibling;
+
+    checkboxes[i].addEventListener('click', () => {
+      if (checkboxes[i].checked) {
+        task.style.textDecoration = 'line-through';
```

Comment:
- [ ] It should have implement `change` instead of `click` event-listener on the checkbox.